integration_tests/test_nft_transfer.py

```diff
@@ -0,0 +1,211 @@
+import json
+import time
+from pathlib import Path
+
+import pytest
+
+from .ibc_utils import start_and_wait_relayer
+from .utils import cluster_fixture
+
+pytestmark = pytest.mark.ibc
```

Comment:
not sure if we should mark nft transfer separately

Reply:
I'm also not sure. We can change this later if tests take too much time on CI.